nemo/collections/duplex_s2s/models/duplex_s2s_model.py

```diff
@@ -599,8 +564,6 @@ def configure_model(self) -> None:
                     "mlp.gate_proj": ColwiseParallel(),
                     "mlp.up_proj": ColwiseParallel(),
                     "mlp.down_proj": RowwiseParallel(output_layouts=Shard(1)),
-                    # "pre_feedforward_layernorm": SequenceParallel(),
-                    # "post_feedforward_layernorm": SequenceParallel(),
```

Comment:
Please keep this for awareness for now, some HF models have these extra layers, we'll need a more generic implementation here to support this.